app/repodata.schema.v0.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Wondering how this file will be handled once the Pyodide kernel is moved to a separate repo? (https://github.com/jupyterlite/jupyterlite/pull/854)

Reply:
It only gets used by `jupyter lite check`, and wouldn't _strictly_ need to be shipped with the app (or, now, extension).

We'd also move the changed chunks of `litePLuginSettings` out of the main schema into the dedicated `check`... I _do_ think it's worth shipping the main schema with the as-deployed app.